libs/wire-api-federation/test/golden/testObject_LeaveConversationResponse1.json

```diff
@@ -1,3 +1,3 @@
 {
-    "Right": {}
+    "Right": []
```

Comment:
Should remain an object! 👿

Reply:
This is fine. It is a federated call, so no clients involved that could break.